smarty/sampler.py

```diff
@@ -95,6 +95,9 @@ def __init__(self, molecules, basetypes_filename, initialtypes_filename, decorat
         # Define internal typing tag.
         self.typetag = 'atomtype'
 
+        # Save bond list to use throughout
+        self.bondset = [("-","simply"), ("=", "doubly"), ("#","triply"), (":", "aromatic"), ("~","any")]
```

Comment:
I think the bond set should be general so we can use it throughout later if we need to